### DISCUSSION TITLE
Job Board

### DISCUSSION THREAD

**Discussion by loftwah** [EddieHubCommunity/LinkFree#6801] [Ideas]
Recruiters or businesses can display job openings and collect applications.

**Comment by shoebilyas123** [comment]
Love this idea <3

**Comment by zaid-ahmad** [comment]
This is great...
but no jobs posted here yet lol

**Reply by loftwah** [reply]
This isn't the job board, this is a discussion for creating a job board in our application.

**Reply by zaid-ahmad** [reply]
Whoops, misunderstood. My bad..

**Reply by waellerbe** [reply]
@loftwah : In keeping with the discussion, I would like to stay in contact with this topic. As I learn to participate in various discussions, I will be able to have encouraging insight to share with professionals who may be new to GitHub and **EddieHub.**

**Comment by eddiejaoude** [comment]
I really like this idea also, @SaraJaoude has started planning this also as part of EddieHub not LinkFree - we need some thought on where to put this. But maybe having it on LinkFree is a good idea. Thanks for the suggestion 👍
